src/goto-instrument/reachability_slicer.cpp

```diff
@@ -240,6 +243,32 @@ void reachability_slicer(
   s(goto_model.goto_functions, p, include_forward_reachability);
 }
 
+/// Perform reachability slicing on goto_model for selected functions.
+/// \param goto_model Goto program to slice
+/// \param functions The functions relevant for the slicing (i.e. starting
+/// point for the search in the cfg). Anything that is reachable in the CFG
+/// starting from these functions will be kept.
+void function_path_reachability_slicer(
+  goto_modelt &goto_model,
+  const std::string &functions_list)
+{
+  reachability_slicert s;
```

Comment:
s->slicer?

Reply:
This (and the following) were renamed as suggested. Thanks.